### PR TITLE
Add dappnode peer to production IPFS node

### PR DESCRIPTION
**Pull request by mateumiralles**
Updating `addDappnodePeerToLocalIpfsNode()` to `addDappnodePeerToIpfsNodes()` to also add the dappnode peer to production the production IPFS node.

### CODE REVIEW

File: packages/migrations/src/addDappnodePeerToIpfsNodes.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "167.86.114.131", //local node
+    "65.109.51.31", //production node
+  ];
+
+  const ipfsAlias = "ipfs.dappnode";
+  const dappnodeIpfsPeer = (ipfsNodeIp: string) =>
+    `/ip4/${ipfsNodeIp}/tcp/4001/p2p/QmfB6dT5zxUq1BXiXisgcZKYkvjywdDYBK5keRaqDKH633`;
+
+  // forEach node adding
+  nodesIps.forEach(async (node) => {
```
Comment (pablomendezroyo) [inline packages/migrations/src/addDappnodePeerToIpfsNodes.ts:15]: Using forEach with asynchronous operations can lead to unexpected behaviors and difficulties in managing control flow. Please use other control flow like for const of or Promise.all

File: packages/migrations/src/addDappnodePeerToIpfsNodes.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { logs } from "@dappnode/logger";
+
+//Adds Dappnode IPFS peers to the local IPFS node for more stability when fetching IPFS content
+export async function addDappnodePeerToIpfsNodes() {
+  const nodesIps = [
+    "167.86.114.131", //local node
```
Comment (pablomendezroyo) [inline packages/migrations/src/addDappnodePeerToIpfsNodes.ts:6]: This is not local but remote - dev/testing node

File: packages/migrations/src/addDappnodePeerToIpfsNodes.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "65.109.51.31", //production node
```
Comment (pablomendezroyo) [inline packages/migrations/src/addDappnodePeerToIpfsNodes.ts:7]: remote production

File: packages/migrations/src/addDappnodePeerToIpfsNodes.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  ];
+
+  const ipfsAlias = "ipfs.dappnode";
+  const dappnodeIpfsPeer = (ipfsNodeIp: string) =>
+    `/ip4/${ipfsNodeIp}/tcp/4001/p2p/QmfB6dT5zxUq1BXiXisgcZKYkvjywdDYBK5keRaqDKH633`;
```
Comment (pablomendezroyo) [inline packages/migrations/src/addDappnodePeerToIpfsNodes.ts:12]: Are you sure it works like this? each IPFS node has a different ID, so production and dev nodes have a different node ID 

Checkout https://docs.ipfs.tech/reference/kubo/rpc/#api-v0-swarm-peering-add

**Review by pablomendezroyo — CHANGES_REQUESTED** [review]

**Review by pablomendezroyo — APPROVED** [review]

**Comment by 3alpha** [comment]
Pls do not add production node here, dev is enough.

**Comment by pablomendezroyo** [comment]
Not required :(

**Pull request closed without merging**